src/rhsm/certificate.py

```diff
@@ -563,7 +563,7 @@ def write(self, pem_path):
         f.write(self.content)
         self.path = pem_path
         f.close()
-        os.chmod(pem_path, 0o600)
+        os.chmod(pem_path, 0o644)
```

Comment:
Why not just drop the line and allow umask to take over?  (Like for the Certificate part: https://github.com/candlepin/subscription-manager/blob/c0f3f166ecfd731a2c68d9290750a7e9108dd8f9/src/rhsm/certificate.py#L246-L259 )

Reply:
@jamescassell administrator of the system can set umask for root user something different than default value is.

Reply:
seems it would be prudent to make the same change in the other location to be consistent.  (If I have umask 077, I'll now have the `-key.pem` key file as 644 and the `.pem` certificate file as 600.)

Reply:
> seems it would be prudent to make the same change in the other location to be consistent. (If I have umask 077, I'll now have the `-key.pem` key file as 644 and the `.pem` certificate file as 600.)

Though it might be interesting to just set umask at the start of the process.  (Interesting discussion here: https://bugs.python.org/issue29214 )